src/System Application/Test/Data Classification/src/DataClassificationMgtTests.Codeunit.al

```diff
@@ -549,7 +544,7 @@ codeunit 135150 "Data Classification Mgt. Tests"
         DataSensitivity.DeleteAll();
 
         // [GIVEN] A table number
-        TableNo := 1180;
+        TableNo := Database::"Data Sensitivity";
```

Comment:
I somewhat get changing 18 or 27 or 1180 to some table(s) that do exist, if that then doesn't influence the behaviour of the test... but shouldn't the replacements be as unique? it seems this PR is changing (almost) everything to Data Sensitivity... Does it matter that all these tests used different values before and now all use the same?

Reply:
The change to `TableNo` is related to the change in `DataClassificationMgtImpl.Codeunit.al`:

I added the logic `procedure IsSupportedTable`. If the table doesn't exit, it will return `false`. Table 18, 27... do not exist in the context of the test, so they will not be inserted into the `Data Sensitivity` table, then causing the tests to fail.

The reason behind the change is: `Data Privacy Utilities` (using configuration package) [only support TableType = Normal](https://dynamicssmb2.visualstudio.com/Dynamics%20SMB/_git/NAV?path=/App/Layers/W1/BaseApp/System/RapidStart/ConfigValidateManagement.Codeunit.al&version=GBmaster&line=949&lineEnd=949&lineStartColumn=21&lineEndColumn=41&lineStyle=plain&_a=contents), so we will align this with `Data Classification Worksheet`.

Reply:
just out of curiosity, what happens when i install an app with a table, classify it, then uninstall the app? The data remains in our system, the classification should too... in case the app is ever re-installed... or something else...

Reply:
Good question.

The Data Classification will stay in our system. We do not delete data in `Data Sensitivity` table. Unless it's unclassified, (this is done via `SyncAllFields`, it can be triggered by an action on the `Data Privacy Utilities` page, it's called Add New Fields or something)

```
/// <summary>
/// Synchronizes the Data Sensitivity table with the Field table. It inserts new values in the Data Sensitivity table for the
/// fields that the Field table contains and the Data Sensitivity table does not and it deletes the unclassified fields from
/// the Data Sensitivity table that the Field table does not contain.
/// </summary>
procedure SyncAllFields()
```

Reply:
Good question.

The Data Classification will stay in our system. We do not delete data in `Data Sensitivity` table. Unless it's unclassified, (this is done via `SyncAllFields`, it can be triggered by an action on the `Data Privacy Utilities` page, it's called Add New Fields or something)

```
/// <summary>
/// Synchronizes the Data Sensitivity table with the Field table. It inserts new values in the Data Sensitivity table for the
/// fields that the Field table contains and the Data Sensitivity table does not and it deletes the unclassified fields from
/// the Data Sensitivity table that the Field table does not contain.
/// </summary>
procedure SyncAllFields()
```